homeassistant/components/velbus/entity.py

```diff
@@ -35,3 +40,26 @@ async def async_added_to_hass(self) -> None:
 
     async def _on_update(self) -> None:
         self.async_write_ha_state()
+
+
+_T = TypeVar("_T", bound="VelbusEntity")
+_P = ParamSpec("_P")
+
+
+def cmd(
```

Comment:
```suggestion
def api_call(
```
Maybe just a slightly better name than just `cmd`